Plugins/Creature/Creature.cpp

```diff
@@ -2850,9 +2850,12 @@ NWNX_EXPORT ArgumentStack DoItemCastSpell(ArgumentStack&& args)
           ASSERT_OR_THROW(delay >= 0.0f);
         auto projectilePathType = args.extract<int32_t>();
         auto projectileSpellID = args.extract<int32_t>();
+        auto oidItem = args.extract<ObjectID>();
+        auto impactScript = args.extract<std::string>();
```

Comment:
Like your other PR, this is also a breaking ABI change, would be nice to mention it in the changelog.

Reply:
Ah, my bad! I didn't realize that would be ABI breaking. Can I blame inexperience? I'll note it in the changelog, same with the other PR.